src/components/GameTabs.vue

```diff
@@ -93,6 +98,22 @@ const taskCompletedMessage = computed(() => {
   return t('gameTabs.taskCompleted');
 });
 
+const updateVideoStarted = async (taskId) => {
+  try {
+    await authStore.roarfirekit.updateVideoMetadata(selectedAdmin.value.id, taskId, 'started');
+  } catch (e) {
+    console.log('Error while updating video completion', e);
```

Comment:
Catching and then not throwing anything will "swallow" this error. This might be what we want, but I just want to make sure. Is the intention to swallow this error in order to prevent it from disrupting gameplay?

In any case, maybe we want to use `console.error` instead.